pkg/preflights/host-preflight.yaml

```diff
@@ -355,4 +363,24 @@ spec:
               message: "Neither 'nameserver localhost' nor 'nameserver 127.0.0.1' can be present in resolv.conf. Remove them to continue."
           - pass:
               when: "false"
-              message: "Neither 'nameserver localhost' nor 'nameserver 127.0.01' is present in resolv.conf"
\ No newline at end of file
+              message: "Neither 'nameserver localhost' nor 'nameserver 127.0.01' is present in resolv.conf"
```

Comment:
This seems to have replaced my changes to merge these two resolv.conf preflights into one. Copy and paste from here to fix it. https://github.com/replicatedhq/embedded-cluster/blob/main/pkg/preflights/host-preflight.yaml#L348-L358